src/test/regress/expected/recurring_outer_join.out

```diff
@@ -0,0 +1,1561 @@
+CREATE SCHEMA recurring_outer_join;
+SET search_path TO recurring_outer_join;
+SET citus.next_shard_id TO 1520000;
+SET citus.shard_count TO 5;
```

Comment:
for outer joins, shard count is an interesting param. Sometimes, some bugs might only show up with certain shards have duplicate non-dist key rows.

In that regard, I'd be fine to make shard count 16 (or 32) in this test